server/views/layout-react.jade

```diff
@@ -11,5 +11,12 @@ html(lang='en')
     script!= state
     script.
       window.webpackManifest = !{JSON.stringify(chunkManifest || {})};
+    script.
+      if (typeof localStorage !== 'undefined') {
+        const fccTheme = JSON.parse(localStorage.getItem('fccTheme'));
```

Comment:
`JSON.parse()` required for the manner in which `store` 'stores' values in `localStorage`

See https://github.com/marcuswestin/store.js/issues/174